build/BuildSteps.cs

```diff
@@ -341,7 +341,7 @@ public static void Test()
 
             Environment.SetEnvironmentVariable("DURABLE_FUNCTION_PATH", Settings.DurableFolder);
 
-            Shell.Run("dotnet", $"test {Settings.TestProjectFile} -f net8.0 --logger trx");
+            Shell.Run("dotnet", $"test {Settings.TestProjectFile} -f net8.0 --logger trx --filter \"{TestingTraits.TraitName.Category}~{TestingTraits.TestCategory.FinalIntegrationTests}\"");
```

Comment:
Why do we need to add traits? And why would it be called "final"

Reply:
so we need to add traits because we want to test the E2E functionality for when we have the core tools OOP host as the root and inproc6 and inproc8 host's as child folders. Before I had integration tests in the `feature/oop-host` branch where we try to run the inproc6/inproc8 host runtimes, but we ultimately fail since the artifact hasn't been assembled with the child folders. Technically those are still integration tests (but they fail since the artifacts have not been generated).

So that's why I added the name `FinalIntegrationTests`. Please let me know if you have better suggestions for the name haha

Reply:
maybe `FinalArtifactsIntegrationTests` would be more clear?

Reply:
I think `FinalArtifactsIntegrationTests` is better; or even just `ArtifactsIntegrationTests` since these are integration tests that will run with artifacts

Reply:
Changed the name to be `RequiresNestedInProcArtifacts`